rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -33,13 +29,7 @@ There are many systems available for employees to use to complete an expense rei
 * Integration capabilities with Accounting Software
 
 
-Regardless of the system, the bulk of the process should be the same across most modern systems.
-
-
-
-At SSW we find the greatest overall value for expense reimbursements is to use the Xero Expense app and the Xero accounting system in combination. 
-
-
+Based on the simplicity in use and overall satisfaction of the Xero Expense app, it is recommended as a leading expense app to use in combination with the Xero Accounting Software. 
```

Comment:
"It is recommended to use the Xero Expense app in tandem with the Xero Accounting software because this combination makes it simple to perform reimbursements and has a high level of customer satisfaction."

I think this wording is better, what do you think?

Reply:
Yes, I agree.